assertj-core/src/test/java/org/assertj/core/internal/bigdecimals/BigDecimalAssert_precision_Test.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * Copyright 2012-2023 the original author or authors.
+ */
+package org.assertj.core.internal.bigdecimals;
+
+import org.junit.jupiter.api.Test;
+
+import java.math.BigDecimal;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatNullPointerException;
+import static org.assertj.core.api.BDDAssertions.then;
+
+class BigDecimalAssert_precision_Test {
```

Comment:
`precision` would be a new navigation method and we defined a base test interface for such cases.

Could you try to add the following?

```suggestion
class BigDecimalAssert_precision_Test implements NavigationMethodBaseTest<BigDecimalAssert<BigDecimal>> {
```